operate/resource.py

```diff
@@ -124,3 +128,16 @@ def store(self) -> None:
             ),
             encoding="utf-8",
         )
+
+        self.load(path)  # Validate before making backup
+
+        for i in reversed(range(N_BACKUPS - 1)):
+            older = path.with_name(f"{path.name}.bak.{i}")
+            newer = path.with_name(f"{path.name}.bak.{i + 1}")
```

Comment:
why not save with some epoch or timestamp?

Reply:
It will make the logic more complex. Keeping the latest 5 successful storages seems will help us resolve most of the issues we have with missing config files.

Reply:
Also, file time can be read from the OS.

Reply:
Good point! My only reasoning was that it can help by comparing the timestamps in logs, to give more clues where a problem can exist. But yeah, OS time can also help.